xml/core_registers.xml

```diff
@@ -83,9 +83,11 @@
 
             2: The Trigger Module caused a breakpoint exception. (priority 4, highest)
 
-            3: The debugger requested entry to Debug Mode. (priority 2)
+            3: The debugger requested entry to Debug Mode. (priority 1)
 
-            4: The hart single stepped because \Fstep was set. (priority 1, lowest)
+            4: The hart single stepped because \Fstep was set. (priority 0, lowest)
+
+            5: The hart halted directly out of reset. (priority 2)
```

Comment:
is it OK to report `3` in this case? otherwise this is not a backwards compatible change, and I can imagine this is hard to implement in all cases.

Reply:
That's a good point. 3 should be acceptable.